### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by chris-little**
More user friendly repo ReadMe.md incorporating @solson-nws proposed text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,57 +6,91 @@ This is the GitHub repository of the [OGC Environmental Data Retrieval API Stand
...
+### EDR API Query Patterns
+The EDR API allows a query, constructed of a spatio-temporal pattern, to retrieve data just for that pattern, from a data collection resource. The main patterns are:
+- **1 - Position**: Retrieve data for point/position for several parameters at a time instant, or as a timeseries, or as a vertical profile for a time instant.
+- **2 - Area**:  Retrieve data within a polygon or rectangular tile/subset at a time instant, or as a timeseries, or as a vertical profile for a time instant.
+- **3 - Trajectory and Corridor**: Retrieve data along a 2D, 3D or 4D trajectory or within a defined corridor around a trajectory.
 
-**A public Hackathon/Sprint** was held [virtually in March 2020](https://github.com/opengeospatial/EDR-API-Sprint) and another was held [9-10 November 2020](https://github.com/opengeospatial/OGCAPI-EDR-Sprint2) to help finalise the specification. There was a **public Webinar** outlining the Sprint's objectives on Wednesday 4 November 2020.
+Each of these patterns is optional, but a compliant API should implement at least one of them.
```
Comment (m-burgoyne) [inline README.md:15]: I think this statement should be moved before any of the query patterns are listed (see above)

File: README.md [hunk 2]
```diff
@@ -6,57 +6,91 @@ This is the GitHub repository of the [OGC Environmental Data Retrieval API Stand
 
 The **OGC API - Environmental Data Retrieval** candidate standard is part of the OGC API suite of standards. [OGC API standards](https://ogcapi.ogc.org) define modular API building blocks to spatially enable Web APIs in a consistent way. [OpenAPI](http://openapis.org) is used to define the reusable API building blocks.
 
-**The public comment period on the EDR API closed September 28th.** [Go here for more information.](https://www.ogc.org/standards/requests/215)
+### EDR API Query Patterns
+The EDR API allows a query, constructed of a spatio-temporal pattern, to retrieve data just for that pattern, from a data collection resource. The main patterns are:
```
Comment (m-burgoyne) [inline README.md:10]: I Think this line should be changed to:

The EDR API allows a query, constructed of a spatio-temporal pattern, to retrieve data just for that pattern, from a data collection resource. **Each of these patterns is optional, but a compliant API must implement at least one.**
The main patterns are:

File: README.md [hunk 3]
```diff
@@ -6,57 +6,91 @@ This is the GitHub repository of the [OGC Environmental Data Retrieval API Stand
...
 
-Summarizing these sections. EDR Query Patterns will be:  
-- **1 - Position**: Retrieve data for point/position or identified location with n parameters at a time instant, or a timeseries, or a vertical profile for a time instant.
-- **2 - Area**:  Retrieve data within a polygon or rectangular tile/subset at a time instant, or a timeseries, or a vertical profile for a time instant.
-- **3 - Trajectory and Corridor**: Retrieve data along a 2D, 3D or 4D trajectory or within a defined corridor around a trajectory.
+**The public comment period on the EDR API closed 28th September 2020.** [Go here for more information.](https://www.ogc.org/standards/requests/215)
 
-- **4 - Ensembles**: Support for explicitly related ensembles of EDR resources. [Note: This will be in a later version]
```
Comment (m-burgoyne) [inline README.md:45]: Delete this line

**Review by m-burgoyne — CHANGES_REQUESTED** [review]
Suggest moving the query patterns are optional text and removing the reference to ensemble queries

File: README.md (hunk 2 above)
Comment (chris-little) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (chris-little) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (chris-little) [reply README.md:45]: Done

**Review by solson-nws — APPROVED** [review]
Chris:  I approve, but suggest a few minor tweaks.  First, future work is buried.  You have to click on "minutes, actions and notes" in order to see future work.  I suggest pulling that to the front.  Second, should we indicate our plans for modifications to the current spec now in the readme?  That might make others more comfortable voting "yes" to the standard.  And finally, I don't see the reference to implementations and sample queries with explanation.  I think that piece is critical to helping understanding.  We may also want to let folks know that Unidata plans to have an EDR implementation in place during 2021.  There are probably others we want to cite as well.  Hope this makes sense, and let me know if you have any questions.
Steve

**Comment by chris-little** [comment]
Addressed reviewers' comments wiwth simple changes. Will merge for the OGC Apache OSGeo hackathon.

**Pull request merged**